### PR TITLE
updated doc for key type and size

### PR DESCRIPTION
**Pull request by SaumyaBhushan**
xref https://github.com/kubernetes/kubeadm/issues/2928

### CODE REVIEW

File: content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md
Comment (neolit123) [inline content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md:222]: please add new note instead of deleting the old one.

/sig cluster-lifecycle

**Comment by neolit123** [comment]
from the PR description remove:
Fixes https://github.com/kubernetes/kubeadm/issues/2928

or change it to:
xref https://github.com/kubernetes/kubeadm/issues/2928

**Review by neolit123 — COMMENTED** [review]
/lgtm

thanks
SIG Docs approvers can only merge this change.

File: content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md
Comment (SaumyaBhushan) [reply content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md:225]: Hey @neolit123 ! Please have a just pushed the latest changes . Does this make sense ?

**Comment by aj11anuj** [comment]
Hello @SaumyaBhushan, netlify checks have failed due to some errors. You need to rebase against main or force push a commit. So, that failed netlify checks can again be started. It's required that all netlify checks must have passed before merging your PR.

**Comment by windsonsea** [comment]
You can try the following commands to trigger the CI again and keept the `/lgtm` label:

```
git reset
git commit --amend --no-edit
git push origin doc-update -f
```

**Review by Arhell — COMMENTED** [review]
Please re-puch commit to restart netlify

**Comment by SaumyaBhushan** [comment]
Thanks everyone for your suggestions . I need to rebase it .

**Comment by SaumyaBhushan** [comment]
@aj11anuj  @windsonsea @Arhell . Please review it now .

**Review by aj11anuj — APPROVED** [review]
Ready to get merged 👍,

Maybe i won't be able to put `lgtm` label, as it requires membership and currently i didn't have membership. Still let's try
/lgtm

**Comment by neolit123** [comment]
> Once this PR has been reviewed and has the lgtm label, please assign [divya-mohan0209](https://github.com/divya-mohan0209) for approval. For more information see [the Kubernetes Code Review Process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process).

/lgtm
/assign @divya-mohan0209

**Comment by neolit123** [comment]
please leave more time for the maintainers to react.
if you are not getting an approval by the end of next week, you can ask in #sig-docs in k8s slack.

File: content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md
Comment (reylejano) [reply content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md:226]: I agree it looks odd to have two notes. I agree that putting the suggested note in this PR as plain text before the existing note

Comment (SaumyaBhushan) [reply content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md:226]: Sure thing !

Comment (tengqm) [reply content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md:225]: @SaumyaBhushan I think there is a misunderstanding here.
The secret and the key expires after two hours. This is something we want to highlight to the users, thus a `note` shortcode is warranted.
However, the key is a AES key of 32 bytes ... this doesn't have to be a note. I may and may not care about this details, because it doesn't hurt if I am not aware of it.
Make sense?

Comment (tengqm) [reply content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md:225]: So ... I am sorry if I was not clear in previous comments.
The note decorator/shortcode is supposed to be applied on the expiration statement because that is sometime not well received and causing failures.
However, the size of the key is not that important and it doesn't warrants a note. 

```
  The certificate key is a hex encoded string that is an AES key of size 32 bytes.

  {{< note >}}
  The `kubeadm-certs` Secret and decryption key expire after two hours.
  {{< /note >}}
```

Comment (neolit123) [reply content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md:225]: @SaumyaBhushan would you be able to update this again? ^^^^

Comment (SaumyaBhushan) [reply content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md:225]: Sure , I can . Sorry I didnt get time on week days to correct it .

**Pull request merged**
